### PR TITLE
[4.0] [a11y] Alert color change to pass WCAG AA

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue # .

### Summary of Changes
Some alerts fail WCAG AA. This style change allows them to pass.

### Testing Instructions
Apply patch and navigate admin, checking alerts.

alert

### CODE REVIEW

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 6b718badae032151c5e827621a838eb92a035847<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14813">issues.joomla.org/tracker/joomla-cms/14813</a>.</sub>

**Comment by dgt41** [comment]
@ciar4n maybe a total override of the default bootstrap variables is easier
Also you committed you configuration here, you need to revert that :)

File: configuration1.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<?php
+class JConfig {
+	public $offline = '0';
```
Comment (dgt41) [inline configuration1.php]: you need to revert this...

Comment (ciar4n) [inline configuration1.php]: I'm an idiot.. thank you!)

**Comment by ciar4n** [comment]
@dgt41 Yep. In this PR I override the mixin which creates the alert. I agree if we continue making such changes a total override is the way to go. Currently everything added in ../administrator/templates/atum/scss/vendor/bootstrap/ is extra CSS to load.

**Review by dgrammatiko — APPROVED** [review]

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 46698967284a99eead27c51f57795bdb62096cc7<br /><br />These colour combos pass the a11y standards - I will leave the comment of @dgt41 to others who understand the code aspect better ;)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14813">issues.joomla.org/tracker/joomla-cms/14813</a>.</sub>

**Review by C-Lodder — APPROVED** [review]

**Comment by franz-wohlkoenig** [comment]
RTC as there are 2 successfully Tests?

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14813">issues.joomla.org/tracker/joomla-cms/14813</a>.</sub>

**Comment by C-Lodder** [comment]
@wilsonge - please merge

**Pull request merged**